packages/vue/src/components/atoms/SfTexarea/SfTextarea.spec.js

```diff
@@ -1,8 +1,22 @@
 import { shallowMount } from "@vue/test-utils";
 import SfTextarea from "./SfTextarea.vue";
 describe("SfTextarea.vue", () => {
-  it("renders a component", () => {
-    const component = shallowMount(SfTextarea);
-    expect(component.classes("sf-textarea")).toBe(true);
+  it("renders component with defaults", () => {
+    const wrapper = shallowMount(SfTextarea);
+    expect(wrapper.classes("sf-textarea")).toBe(true);
+  });
+  it("renders component with value", () => {
+    const wrapper = shallowMount(SfTextarea, {
+      propsData: {
+        value: "text",
+      },
+      listeners: {
+        input: () => {
+          expect(component.vm.$data.input).toBeTruthy();
```

Comment:
There is component error here: ``` [Vue warn]: Error in v-on handler: "ReferenceError: component is not defined"```
IT is functional component so I think putting whole object whir props and listeners into `context` and change `prosData` in `props` should help: 
```
context: {
        props: {
          value: "text",
        },
        listeners: {
          input: () => {
            expect(component.vm.$data.input).toBeTruthy();
            done();
          },
        },
      }
```